src/sidebar/services/frame-sync.js

```diff
@@ -252,6 +252,7 @@ export default function FrameSync($rootScope, $window, store, bridge) {
    * @param {string[]} tags
    */
   this.focusAnnotations = function (tags) {
+    store.focusAnnotations(tags);
```

Comment:
Previously, we were only partially keeping the `focusedAnnotation` state updated...